### PR TITLE
add tests for #984

### PR DESCRIPTION
**Pull request by eroux**
corresponds to https://github.com/gregorio-project/gregorio/pull/1000

### CODE REVIEW

**Comment by eroux** [comment]
PR has changed: https://github.com/gregorio-project/gregorio/pull/1001

File: tests/tex-output/bugs/fix-984/fix-984.tex [hunk 1]
```diff
@@ -0,0 +1,12 @@
+\documentclass[12pt, a5paper]{book}
+\usepackage{libertine}
+\usepackage[autocompile]{gregoriotex}
+\begin{document}
+\gresetbarspacing{new}
+\gresetbreakbeforeeuouae{ragged}
+\gresetlastline{ragged}
+
+\gregorioscore{fix-984-1}
```
Comment (rpspringuel) [inline tests/tex-output/bugs/fix-984/fix-984.tex]: Did you forget to upload this score?

File: tests/tex-output/bugs/fix-984/fix-984.tex [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+\gregorioscore{fix-984-2}
```
Comment (rpspringuel) [inline tests/tex-output/bugs/fix-984/fix-984.tex]: Same with this one.

**Comment by eroux** [comment]
indeed, thanks

**Pull request merged**